wicket-orientdb/src/main/java/ru/ydn/wicket/wicketorientdb/model/OQueryModel.java

```diff
@@ -166,35 +162,22 @@ public void clearFilterCriteriaManagers() {
         queryManager.clearFilterCriteriaManagers();
     }
 
-    /**
-     * Set value for context variable
-     * @param varName name of the variable to set
-     * @param value {@link IModel} for the variable value
-     * @return this {@link OQueryModel}
-     */
-    @SuppressWarnings("unchecked")
-	public OQueryModel<K> setContextVariable(String varName, IModel<?> value)
```

Comment:
The same question - why context variables are being deleted? We using that in Orienteer.